tests/cupyx_tests/scipy_tests/sparse_tests/test_linalg.py

```diff
@@ -1268,3 +1268,28 @@ def test_spilu_0(self, dtype, xp, sp):
         else:
             ainv = sp.linalg.spilu(a)
         return ainv.solve(b)
+
+
+@testing.parameterize(*testing.product({
```

Comment:
Would you test each matrix format: `csr`, `csc`, `coo`?

Reply:
Okay, I'll add matrix format to test.